charts/scroll-proving-sdk/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: v2
+description: scroll-proving-sdk helm charts
```

Comment:
Example chart for showing how to package a service built using the scroll-proving-sdk library